projects/RabbitMQ.Client/client/impl/IncomingCommand.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Buffers;
+
+namespace RabbitMQ.Client.Impl
+{
+    internal readonly struct IncomingCommand : IDisposable
```

Comment:
```
Marshal.SizeOf(typeof(IncomingCommand));
```

shows it is 40 bytes large which technically violates the principle of structs should be around 16-24 bytes. Am I assuming correctly the potential perf impact of that is largely mitigated by passing them around with `in`,`out` or `ref`?

Reply:
Yes, while this struct is passed around, the copying overhead is neglected by the `in` passing.